### ISSUE TITLE
instant machines auto-connect function

### ISSUE THREAD

**Issue by deimosx6** [Algalish/SmartIssueTracker#155]
please add auto-connect to instant machines

**Comment by Algalish** [comment]
what are instant machines? a mod?

**Comment by LegionsNecro666** [comment]
Yes Instant Machines are a Mod designed by Andre Aquilla, it allows resources to be instantly made without a crafting timer. Your smart mod works on it with the machines but the auto connect still not working. Thanks again for your hard work and determination for creating the best mod for satisfactory, Smart should be enabled by default in vanilla. Well done.

**Comment by Algalish** [comment]
I am closing all the open issues because old issues may not require any action. 
But if this is still something you need or if this is something that is not working in the latest version in U6 - please reopen the issue and I will look into it.
Thank you
